src/components/bodhi/createTopic.js

```diff
@@ -167,39 +176,69 @@ class CreateTopic extends React.Component {
               placeholder="e.g. Who will be the next president of the United States?"
             />)}
           </FormItem>
-          <FormItem
-            {...formItemLayout}
-            label="Betting End Block"
-          >
-            {getFieldDecorator('bettingEndBlock', {
+
+          {this.createInputNumberField(
+            'bettingStartBlock',
+            'Betting Start Block',
+            {
               rules: [{
-                required: true, message: 'Please enter a future block number.',
+                required: true, message: 'Must be greater than or equal to current block number.',
```

Comment:
[open question] should we prevent the endblock number too close to current/start block num

Reply:
Maybe we can add a warning when we add the date/time picker. But don't think we should restrict the user since its an open platform and shouldn't impose rules that arent necessary.